docs/topics/migrations.txt

```diff
@@ -476,6 +476,30 @@ You can pass a second callable to
 want executed when migrating backwards. If this callable is omitted, migrating
 backwards will raise an exception.
 
+.. note::
+
+    When migrating data across models spanning more than one app, ``dependencies``
+    should include the latest migration of each app that is involved, otherwise you
+    will get an error: "LookupError: No installed app with label '``myappname``'".
```

Comment:
I might be doing something wrong, but I still get the error in the latest master.

I did: git fetch upstream; git checkout master; git merge upstream/master
[switched to my branch] and did: git rebase master

Here's my migration code:

def move_m1(apps, schema_editor):
    m1 = apps.get_model("app1", "M1")
    m1_2 = apps.get_model("app2", "M1")
    for row in m1.objects.all():
        m1_2.create(f1=row.f1)


class Migration(migrations.Migration):

    dependencies = [
        ('app1', '0002_m1_f1'),
        # ('app2', '0001_initial'),
    ]

    operations = [
      migrations.RunPython(move_m1),
    ]

And the error I get:

{{{
Operations to perform:
  Apply all migrations: app1
Running migrations:
  Rendering model states... DONE
  Applying app1.0003_auto_20150129_1705...Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/akulakov/projects/django/django/core/management/__init__.py", line 330, in execute_from_command_line
    utility.execute()
  File "/Users/akulakov/projects/django/django/core/management/__init__.py", line 322, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/akulakov/projects/django/django/core/management/base.py", line 347, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/akulakov/projects/django/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/Users/akulakov/projects/django/django/core/management/commands/migrate.py", line 195, in handle
    executor.migrate(targets, plan, fake=fake, fake_initial=fake_initial)
  File "/Users/akulakov/projects/django/django/db/migrations/executor.py", line 94, in migrate
    self.apply_migration(states[migration], migration, fake=fake, fake_initial=fake_initial)
  File "/Users/akulakov/projects/django/django/db/migrations/executor.py", line 131, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/Users/akulakov/projects/django/django/db/migrations/migration.py", line 111, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/Users/akulakov/projects/django/django/db/migrations/operations/special.py", line 181, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/Users/akulakov/projects/testproj/app1/migrations/0003_auto_20150129_1705.py", line 11, in move_m1
    m1_2 = apps.get_model("app2", "M1")
  File "/Users/akulakov/projects/django/django/apps/registry.py", line 189, in get_model
    return self.get_app_config(app_label).get_model(model_name.lower())
  File "/Users/akulakov/projects/django/django/apps/registry.py", line 147, in get_app_config
    raise LookupError("No installed app with label '%s'." % app_label)
LookupError: No installed app with label 'app2'.
}}}